libs/ui/src/modal.test.tsx

```diff
@@ -31,13 +31,56 @@ describe('Modal', () => {
     render(<Modal content="Do you want to do the thing?" centerContent />);
 
     const modal = screen.getByRole('alertdialog');
+    expect(modal).toMatchInlineSnapshot(`
+      <div
+        aria-label="Alert Modal"
+        aria-modal="true"
+        class="sc-gsDJrp blYTQX ReactModal__Content _"
+        data-testid="modal"
+        role="alertdialog"
+        tabindex="-1"
+      >
+        <div
+          class="sc-hKwCoD RDMJL"
+        >
+          Do you want to do the thing?
+        </div>
+      </div>
+    `);
```

Comment:
I personally don't love using snapshots for HTML content due to the styled-components classnames, which seem to change often. Was there something specific you were trying to test for here?

Reply:
Yes I agree, but libs/ui has a 100% code coverage requirement so it was forcing me to add a test for the Wide modal case, because the css change is underneath the @media(min-width: 480) {} block, it doesn't automatically work with the expect().toHaveStyle() function and I did some googling and found a few threads of people saying its impossible to test things like that. However it does generate a different css classname in the snapshot tests so given the urgency with m13b and the relative unimportance of testing this well that seemed like the easiest thing to do for now to get this change in.

Reply:
Oh that's annoying